global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "9.0.100",
+    "dotnet": "10.0.100-alpha.1.24573.1",
```

Comment:
@Evangelink Do we want to move that fast to 10.x?
I'd also assume that downgrading back to 9.0.100 will be green and avoid the current SB issue until it's investigated further.